frontend/src/forms/subforms/AddressForm.tsx

```diff
@@ -7,10 +7,11 @@ import { ILookupCode } from 'actions/lookupActions';
 import { ILookupCodeState } from 'reducers/lookupCodeReducer';
 import _ from 'lodash';
 import * as API from 'constants/API';
-import { Form, FastSelect, FastInput, Select } from 'components/common/form';
+import { Form, FastInput, Select } from 'components/common/form';
 import { mapLookupCode } from 'utils';
 import { Col } from 'react-bootstrap';
 import { IAddress } from 'actions/parcelsActions';
+import { AutoCompleteText } from 'components/common/form/AutoCompleteText';
```

Comment:
/nit: I'd prefer to see this import grouped together with the other form imports in line 10

Reply:
Oh woops yes, I will do that. Think this was a result of a 'quick-fix'